launcher/ui/widgets/ThemeCustomizationWidget.ui

```diff
@@ -40,40 +40,80 @@
     </widget>
    </item>
    <item row="0" column="1">
-    <widget class="QComboBox" name="iconsComboBox">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
-     </property>
-     <property name="focusPolicy">
-      <enum>Qt::StrongFocus</enum>
-     </property>
-    </widget>
+    <layout class="QHBoxLayout" name="iconsLayout">
+     <item>
+      <widget class="QComboBox" name="iconsComboBox">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="focusPolicy">
+        <enum>Qt::StrongFocus</enum>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QPushButton" name="iconsFolder">
+       <property name="toolTip">
+        <string>View icon themes folder.</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+       <property name="icon">
+        <iconset theme="viewfolder"/>
+       </property>
+       <property name="flat">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+    </layout>
    </item>
    <item row="1" column="0">
-    <widget class="QLabel" name="widgetThemeLabel">
+    <widget class="QLabel" name="widgetStyleLabel">
      <property name="text">
-      <string>&amp;Colors</string>
+      <string>&amp;Widgets</string>
      </property>
      <property name="buddy">
       <cstring>widgetStyleComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
-    <widget class="QComboBox" name="widgetStyleComboBox">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
-     </property>
-     <property name="focusPolicy">
-      <enum>Qt::StrongFocus</enum>
-     </property>
-    </widget>
+    <layout class="QHBoxLayout" name="widgetStyleLayout">
+     <item>
+      <widget class="QComboBox" name="widgetStyleComboBox">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="focusPolicy">
+        <enum>Qt::StrongFocus</enum>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QPushButton" name="widgetStyleFolder">
+       <property name="toolTip">
+        <string>View widget themes folder.</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+       <property name="icon">
+        <iconset theme="viewfolder"/>
+       </property>
+       <property name="flat">
```

Comment:
while it is consistent with the cat thing, I think these two buttons shouldn't be flat... for the cat thing i just abused a button to make a little info hint as it was easier than manually drawing a icon
(can't seem to propose the multiline changes, thx github ui)
![image](https://github.com/PrismLauncher/PrismLauncher/assets/31988415/527dcd66-9eba-4bf5-94cd-249e16eb0e7f)
maybe later on when catpacks we can change the cat tooltip and make a third folder button, then its all consistent again x3

Reply:
catpacks are already merged >:3

Reply:
I think I'll replace the info with a button - then add a checkbox to make the option more self-explanatory

Reply:
I honestly think flat looks better when it's just an icon button

Reply:
I didn't change it. is that okay?